bdns/dns.go

```diff
@@ -502,7 +502,11 @@ func (dnsClient *DNSClientImpl) LookupCAA(ctx context.Context, hostname string)
 			CAAs = append(CAAs, caaR)
 		}
 	}
-	return CAAs, nil
+	response := "CAA query response was empty"
```

Comment:
Since this will be logged in the vast majority of cases, may as well make it economical and have it be either the empty string:

```suggestion
	response := ""
```

Note that the `present: %t` we log suffices to convey the information that there were no applicable CAA records in the response.